it/mail/chrome/messenger/messengercompose/pbMacroSearchOverlay.properties

```diff
@@ -1,4 +1,4 @@
-signatures=FirstName
+signatures=Firme
```

Comment:
In the source EN text I see:
signatures=Signatures
So, I would translate 'Firme' ('Signatures' in Italian)